magento/pwa-studio
# Uses abbreviation for label if english text is not available

## Description

`AN` (Netherlands Antilles) is not returning `full_name_english` for some reason. I have opened a [bug](https://jira.corp.magento.com/browse/MC-35052) with core but we can at least get ahead of it by falling back to the abbreviation for sort order.

## Related Issue





Closes [PWA-673](https://jira.corp.magento.com/browse/PWA-673).

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to cart page or checkout page.
2. In any countries dropdown, verify that the order is correct. Abbreviations will appear at the top, as capital letters are "bigger" than lowercase letters.

## Screenshots / Screen Captures (if appropriate)
[![Image from Gyazo](https://i.gyazo.com/b4b43e8ab74300539716d146b36c8b94.gif)](https://gyazo.com/b4b43e8ab74300539716d146b36c8b94)

## Checklist

* I have updated the documentation accordingly, if necessary.
